osadchyi_homework_1.cpp

```diff
@@ -0,0 +1,65 @@
+#include <cmath>
+#include <iostream>
+
+int main() {
+  int atmpts{3};
+  constexpr short lwr_bndr = 2;
+  constexpr short uppr_bndr = 255;
+
+  std::cout << "Enter three numbers in range " + std::to_string(lwr_bndr) +
+                   "..." + std::to_string(uppr_bndr) +
+                   " separated by space in order a b c.\nIf value is empty "
+                   "leave 0."
+            << std::endl;
+
+  while (atmpts) {
```

Comment:
можно просто для читабельности добавить
```suggestion
  while (atmpts > 0) {
```